scene/gui/tree.cpp

```diff
@@ -5329,10 +5329,12 @@ String Tree::get_tooltip(const Point2 &p_pos) const {
 				pos.x -= get_column_width(i);
 			}
 
+			int buttonOffset = h_scroll->get_max() - h_scroll->get_page() - h_scroll->get_value();
```

Comment:
I'm not entirely sure about the logic yet, but at the very least the variable should be named with the under_score style.